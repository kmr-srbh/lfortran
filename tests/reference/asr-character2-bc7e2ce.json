{
    "basename": "asr-character2-bc7e2ce",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/character2.f90",
    "infile_hash": "97195d2054dbf5075a24b846fcd7dcc115ef93398243c901c04a3951",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-character2-bc7e2ce.stdout",
    "stdout_hash": "1027d25924015220412c5232613501e0caef2accc83b356e8ff80c3d",
    "stderr": "asr-character2-bc7e2ce.stderr",
    "stderr_hash": "a453d5cce85040dc964fdb0ee2b4255629fa29f4b8013603f586a9f0",
    "returncode": 0
}