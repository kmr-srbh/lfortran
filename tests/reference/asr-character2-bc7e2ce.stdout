(TranslationUnit
    (SymbolTable
        1
        {
            character2:
                (Program
                    (SymbolTable
                        2
                        {
                            char1:
                                (Variable
                                    2
                                    char1
                                    []
                                    Local
                                    (StringConstant
                                        "Goat"
                                        (Character 1 4 () PointerString)
                                    )
                                    (StringConstant
                                        "Goat"
                                        (Character 1 4 () PointerString)
                                    )
                                    Save
                                    (Character 1 4 () PointerString)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            char2:
                                (Variable
                                    2
                                    char2
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Character 1 4 () PointerString)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            char_param:
                                (Variable
                                    2
                                    char_param
                                    []
                                    Local
                                    (StringConstant
                                        "App"
                                        (Character 1 3 () PointerString)
                                    )
                                    (StringConstant
                                        "App"
                                        (Character 1 3 () PointerString)
                                    )
                                    Parameter
                                    (Character 1 3 () PointerString)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    character2
                    []
                    [(Assignment
                        (Var 2 char2)
                        (StringConstant
                            "Balle"
                            (Character 1 5 () PointerString)
                        )
                        ()
                    )]
                )
        })
    []
)
